src/components/profile/AvatarCircle.js

```diff
@@ -30,7 +30,6 @@ const FirstLetter = styled(Text).attrs(({ theme: { colors } }) => ({
 `;
 
 export default function AvatarCircle({
-  isAvatarPickerAvailable,
```

Comment:
what about the ShowcaseHeader which uses this to set it to false?

Reply:
only issue i'm seeing

Reply:
right, thanks